applications/app/controllers/InteractiveController.scala

```diff
@@ -5,21 +5,35 @@ import common._
 import contentapi.ContentApiClient
 import conf._
 import conf.switches.Switches
+import model.Cached.RevalidatableResult
 import model._
+import play.api.libs.ws.WSClient
 import play.api.mvc._
 import views.support.RenderOtherStatus
+import conf.Configuration.interactive.cdnPath
+import conf.Configuration.environment.isPreview
 
 import scala.concurrent.Future
 
 case class InteractivePage (interactive: Interactive, related: RelatedContent) extends ContentPage {
   override lazy val item = interactive
 }
 
-class InteractiveController(contentApiClient: ContentApiClient) extends Controller with RendersItemResponse with Logging with ExecutionContexts {
+class InteractiveController(contentApiClient: ContentApiClient, wsClient: WSClient) extends Controller with RendersItemResponse with Logging with ExecutionContexts {
 
   def renderInteractiveJson(path: String): Action[AnyContent] = renderInteractive(path)
   def renderInteractive(path: String): Action[AnyContent] = Action.async { implicit request => renderItem(path) }
 
+  def renderInteractiveServiceworker(path: String): Action[AnyContent] = Action.async { implicit request =>
+    val stage = if (isPreview) "preview" else "live"
+    val serviceWorkerPath = s"$cdnPath/service-workers/$stage/$path/interactive-service-worker.js"
+    wsClient.url(serviceWorkerPath).get().map { response =>
+      Cached(60) {
```

Comment:
if we want to use the cache header, we should parse out the time, and generate a new header from that, to make sure we get all the right surrogate control etc.  Also might be worth putting in a floor of say 5 seconds (or just 404 if it's not present) to make sure it's not missing